tmail-backend/apps/distributed/docker-compose.yml

```diff
@@ -36,6 +36,7 @@ services:
     volumes:
       - ${RSA_PUBLICKEY_PATH}:/root/conf/jwt_publickey # Replace with absolute path to your RSA public key
       - ${RSA_PRIVATEKEY_PATH}:/root/conf/jwt_privatekey # Replace with absolute path to your RSA private key
+      - ./redis.properties:/root/conf/redis.properties # Replace with absolute path to your RSA private key
```

Comment:
Should put in an other folder I think instead of root folder, like we do for some apps on james, like docker-configuration?